deploy-manage/remote-clusters/security-models.md

```diff
@@ -0,0 +1,86 @@
+---
+navigation_title: Security models
+applies_to:
+  deployment:
+    ece: ga
+    ess: ga
+    eck: ga
+    self: ga
+  serverless: unavailable
+products:
+  - id: elasticsearch
+---
+# Remote clusters security models
+
+Remote clusters security models determine how authentication and authorization works between clusters. {{es}} has evolved from a TLS certificate–based model, relying on mutual TLS authentication over the transport interface and duplicated roles across clusters, to a more flexible API key–based model that uses a dedicated service endpoint and supports fine-grained authorization on both local and remote clusters.
```

Comment:
this is a big mouthful. is the evolution of this feature important? 

```
Remote cluster security models determine how authentication and authorization works between clusters. 

There are two models available: 
* [API ..]
* [TLS] (deprecated)

API key [benefits], so use it / migrate to it
```

Reply:
@shainaraskas , I think understanding its evolution helps to:
- understand why all this is so complicated and why the modes are so different
- understand why we have 2 modes (it doesn't make much sense from design point of view aiming for the 2 modes since the beginning).
- understand why the legacy / deprecated mode shouldn't be used

But anyway I'm totally ok removing that part or simplifying it if you think it doesn't add any value.